src/driver/main.cpp

```diff
@@ -59,6 +60,10 @@ struct loader
            ap.set_value(true));
         ap(is_nhwc, {"--nchw"}, ap.help("Treat tensorflow format as nchw"), ap.set_value(false));
         ap(trim, {"--trim", "-t"}, ap.help("Trim instructions from the end"));
+        ap(param_dims,
+           {"--dim_val"},
```

Comment:
This should be `--dim-val`

Reply:
Maybe this should be named `--input-dim` to avoid confusion with dim parameters when we add them later.